src/components/Table/Table.types.ts

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+import { SortingRule, TableOptions, UsePaginationInstanceProps } from 'react-table';
+
+import { DefaultComponentProps } from '../../types';
+
+export interface TableProps<T extends object> extends DefaultComponentProps {
+	options: TableOptions<T>;
```

Comment:
Ik zou eerder de TableProps laten extenden op de options die we nodig hebben (Pick<TableOptions, ...>), props gaan we altijd zo min mogelijk nesten.

Reply:
`TableOptions` slaagt hier op de volledige configuratie van de `react-table` instance, imo is het schaalbaarder om heel deze config te exposen.